content/challenges/medium/4 - Reverse Words/func.js

```diff
@@ -11,5 +11,20 @@
  * @returns {string} The string with the proper words reversed
  */
 module.exports = (str) => {
-    // your code here
+    words = str.split(" ")
+    let rev_words = []
+    for (i = 0; i < words.length; i++) {
+        if (words[i].length >= 5) {
+            let word = words[i]
```

Comment:
check out the `.reverse()` string function. I bet you can do this in less lines :)